### ISSUE TITLE
Issue related to torch.distributed.init_process_group()

### ISSUE THREAD

**Issue by JosephKKim** [shjung13/Standardized-max-logits#9]
Hello thanks for the interesting work!

I am trying to run your code, calcuate_stat.py using the command you provided, 
but the code stops to work torch.distributed.init_process_group. What might be the problem?

Do I have to modify url? or other arguments?

In addition I don't understand how the function calculate_statistics works in detail. Can you give a detailed explanation about the function?
Especially about the if statement where i % 50 == 49.

Thanks in advance!
